overlord/devicestate/handlers_gadget.go

```diff
@@ -47,7 +49,7 @@ func makeRollbackDir(name string) (string, error) {
 	return rollbackDir, nil
 }
 
-func currentGadgetInfo(st *state.State, curDeviceCtx snapstate.DeviceContext) (*gadget.GadgetData, error) {
+func CurrentGadgetInfo(st *state.State, curDeviceCtx snapstate.DeviceContext) (*gadget.GadgetData, error) {
```

Comment:
now that this is exported this require a doc comment, it probably needs to be called CurrentGadgetData ? it had a strange name already, and maybe other functions need to be renamed for consistency

Reply:
Changed